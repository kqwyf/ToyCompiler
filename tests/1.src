_    _123 _abc _1aZ __aa_1_223
a123 A321 Aa90 zZ55 a_A ab__   d1_2
_123 _1aZ zZ55 ab__ d1_2

