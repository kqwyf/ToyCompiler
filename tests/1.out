Token sequence:
_               < IDENTIFIER  , 1      >
_123            < IDENTIFIER  , 2      >
_abc            < IDENTIFIER  , 3      >
_1aZ            < IDENTIFIER  , 4      >
__aa_1_223      < IDENTIFIER  , 5      >
a123            < IDENTIFIER  , 6      >
A321            < IDENTIFIER  , 7      >
Aa90            < IDENTIFIER  , 8      >
zZ55            < IDENTIFIER  , 9      >
a_A             < IDENTIFIER  , 10     >
ab__            < IDENTIFIER  , 11     >
d1_2            < IDENTIFIER  , 12     >
_123            < IDENTIFIER  , 2      >
_1aZ            < IDENTIFIER  , 4      >
zZ55            < IDENTIFIER  , 9      >
ab__            < IDENTIFIER  , 11     >
d1_2            < IDENTIFIER  , 12     >

Symbol table:
1     _
2     _123
3     _abc
4     _1aZ
5     __aa_1_223
6     a123
7     A321
8     Aa90
9     zZ55
10    a_A
11    ab__
12    d1_2
